src/Symfony/Component/Translation/Bridge/Crowdin/README.md

```diff
@@ -23,7 +23,11 @@ where:
 Sponsor
 -------
 
-Help Symfony by [sponsoring][1] its development!
+This bridge for Symfony 5.4/6.0 is [backed][1] by [Crowdin][2].
```

Comment:
Shouldn't this have been 7.1?

Reply:
indeed!

Reply:
fixed in eecb3bb2ae9b7db0c337cbf47e6e0f98822d5375